.github/workflows/make_release.yml

```diff
@@ -0,0 +1,231 @@
+name: Create Release
+on:
+  push:
+    tags:
+      - 'v*'
+env:
+  CIBW_BUILD_VERBOSITY: 2
+  CIBW_TEST_REQUIRES: pytest pooch pytest-localserver pytest-faulthandler
```

Comment:
This comes from `requirements/test.txt` I presume; can we install these from that file in a step, or do they need to be explicitly re-specified here?